File: 30 Days of Code/Day6/C#/string_maluca.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution {
    static void Main(String[] args) {
    
    /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
    int n = int.Parse(Console.ReadLine());

    while (n > 0) {
        string palavra = Console.ReadLine(), impar = "", par = "";
        int tamanho = palavra.Length;
        for (int i = 0; i < tamanho; i++) {
            if (i % 2 == 0) {
                par += palavra[i];
            }
            if (i % 2 != 0) {
                impar += palavra[i];
            }
        }
        Console.WriteLine($"{par} {impar}");
        n--;
    }
    }
}

File: 30 Days of Code/Day6/C/string_maluca.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>


#define MAX 10000

int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    int n;
    scanf("%d", &n);

    while (n > 0) {

        char palavra[MAX], impar[MAX], par[MAX];
        scanf("%s", palavra);

        int tamanho = strlen(palavra);
        int parIndex = 0, imparIndex = 0;

        for (int i = 0; i < tamanho; i++) {
            if (i % 2 == 0) {
                par[parIndex++] = palavra[i];
            } else {
                impar[imparIndex++] = palavra[i];
            }
        }

        par[parIndex] = '\0';
        impar[parIndex] = '\0';

        printf("%s %s\n", par, impar);
        n--;
    }
    return 0;
}
